chore: add skeleton to staking page

iamonuwa opened the pull request:
closes #839

iamonuwa committed:
chore: add skeleton to staking page

iamonuwa committed:
chore: add changelog

iamonuwa committed:
Merge branch 'release/hub' into add-loading-to-staking

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 42f4ec26 | [#3](https://ci.status.im/job/status-web/job/prs/job/linux/job/connector/job/PR-840/3/) | 2025-11-06 17:52:25 | ~45 sec | `connector` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusWeb-connector-251106-175139-42f4ec-pr840-chrome.zip) |
| :heavy_check_mark: | 42f4ec26 | [#3](https://ci.status.im/job/status-web/job/prs/job/linux/job/wallet/job/PR-840/3/) | 2025-11-06 17:52:50 | ~1 min | `wallet` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusWeb-wallet-251106-175139-42f4ec-pr840-chrome.zip) |

JulesFiliot committed:
fix(release): scope remark linting to content directory only

* fix(release): scope remark linting to content directory only

* chore(status.app): add changeset

JulesFiliot reviewed (commented):
All good and nice 👍 
Except the skeleton for the vault's table which is not displaying for some reason:
<img width="2476" height="516" alt="image" src="https://github.com/user-attachments/assets/9689a66e-db3e-4c09-8017-9d5a76732b4c" />
The loading state looks like that:
<img width="1510" height="294" alt="image" src="https://github.com/user-attachments/assets/cc15827a-3977-429f-91fd-e0c30c1fa8af" />

felicio committed:
Rename 'Mobile' to 'New mobile' in routes (#845)

* Rename 'Mobile' to 'New mobile' in routes

why

- https://github.com/status-im/status-website/pull/1620#discussion_r2499897205

* Rename 'Mobile' to 'New mobile' in routes

iamonuwa committed:
refactor: clean up imports and improve vault locking logic

iamonuwa committed:
Merge branch 'release/hub', remote-tracking branch 'origin' into add-loading-to-staking

jkbktl committed:
add withdraw (#813)

* add withdraw

* Add changeset for patch updates

* address feedback

* address feedback

* change contracts address back to production

* add use client to emergencyMode hook

* test fix build

* test fix build

jinhojang6 committed:
feat(layout): add Umami analytics script to RootLayout component

jinhojang6 committed:
fix(layout): update Umami analytics script domains in RootLayout component

felicio committed:
u

jkbktl committed:
update max amount

jkbktl committed:
fix testnet tag and letter-spacing for text-13 class

jkbktl committed:
fix label for app

iamonuwa committed:
Fix vercel comments (#853)

* chore: fix vercel comments

* feat: add Status Wallet detection and update SIWE message creation

* chore: add changelog

* chore: revert create message response

iamonuwa commented:
> If I close Connect Wallet dialog via (X) close button, Stake page gets stuck on loading skeleton.

check again, please.

iamonuwa committed:
chore: point learn more to https://docs.status.network/tokenomics/snt-staking

iamonuwa committed:
chore: refactor stake amount input properly

iamonuwa committed:
chore: keep stake-amount-input dry

iamonuwa commented on apps/hub/src/app/_components/stake/stake-form.tsx:
```diff
@@ -0,0 +1,300 @@
+          register={form.register}
+          isConnected={status === 'connected'}
+          isLoading={isLoading}
+        />
```
https://github.com/status-im/status-web/pull/840/commits/1b32b4f7601e252cd3b47cbc203afa602b64c639

iamonuwa committed:
Merge branch 'release/hub' into add-loading-to-staking

jkbktl commented on apps/hub/src/app/_components/vaults/table-columns.tsx:
```diff
@@ -303,84 +305,92 @@ export const createVaultTableColumns = ({
-                      <span className="whitespace-nowrap">Lock</span>
+                    <Button
+                      variant="primary"
+                      size="32"
```
why are you changing all buttons to `size=32` again?

jkbktl commented on apps/hub/src/app/_components/vaults/vaults-table.tsx:
```diff
@@ -116,14 +118,88 @@ function TableFooter({ table }: TableProps) {
+        <table className="w-full border-collapse">
+          <thead className="h-[40px] border-b border-solid border-neutral-10 bg-neutral-5">
+            <tr>
+              <th className="box-border px-[12px] text-left">
```
```suggestion
              <th className="box-border px-3 text-left">
```

iamonuwa replied:
https://github.com/status-im/status-web/pull/840/commits/d2d4ac8cb8734de737af2b5311f4f900194d0e44

jkbktl replied:
@iamonuwa this is multiple times in the file and needs to be fixed everywhere

jkbktl commented on apps/hub/src/app/_components/vaults/table-columns.tsx:
```diff
@@ -303,84 +305,92 @@ export const createVaultTableColumns = ({
+                  {!emergencyModeEnabled && (
+                    <WithdrawVaultModal
+                      open={isWithdrawModalOpen}
+                      amountWei={row.original.data?.depositedBalance || 0n}
```
This changes Withdraw functionality

iamonuwa replied:
https://github.com/status-im/status-web/pull/840/commits/d2d4ac8cb8734de737af2b5311f4f900194d0e44

iamonuwa committed:
fix: fix emergency withdraw mode check

iamonuwa commented on apps/hub/src/app/_components/vaults/vaults-table.tsx:
```diff
@@ -116,14 +118,88 @@ function TableFooter({ table }: TableProps) {
+    <div className="max-h-[600px] overflow-auto">
+      <div className="min-w-[800px]">
+        <table className="w-full border-collapse">
+          <thead className="h-[40px] border-b border-solid border-neutral-10 bg-neutral-5">
```
https://github.com/status-im/status-web/pull/840/commits/d2d4ac8cb8734de737af2b5311f4f900194d0e44

iamonuwa commented on apps/hub/src/app/_components/vaults/table-columns.tsx:
```diff
@@ -303,84 +305,92 @@ export const createVaultTableColumns = ({
+                        variant="danger"
+                        size="32"
+                        disabled={!isConnected || !isSignedIn}
+                        className="min-w-fit bg-danger-50 text-13 text-white-100 hover:bg-danger-60"
```
https://github.com/status-im/status-web/pull/840/commits/d2d4ac8cb8734de737af2b5311f4f900194d0e44

iamonuwa committed:
chore: replace px with tailwind measurement

JulesFiliot reviewed (approved):
If not seeing the skeleton on the vaults' table is expected, all good ! :shipit:
